hw_diag/templates/device_info.html

```diff
@@ -56,48 +56,69 @@ <h4 class="text-center mb-4">Overall Status:
            <tr>
              {% if diagnostics and diagnostics.BA and diagnostics.BA.endswith('-c') %}
              <td><span class="uil uil-refresh icon"></span> Firmware Version</td>
-             <td class="text-right"><i class="fa fa-star" aria-hidden="true"></i> Premium - {{ diagnostics.FW }} ({{ diagnostics.firmware_short_hash }})</td>
+             <td class="text-right">
+              <span class="badge badge-light p-2"><i class="fa fa-star" aria-hidden="true"></i> Premium - {{ diagnostics.FW }} ({{ diagnostics.firmware_short_hash }})</span>
+              </td>
              {% else %}
              <td><span class="uil uil-refresh icon"></span> Firmware Version</td>
-             <td class="text-right">{{ diagnostics.FW }} ({{ diagnostics.firmware_short_hash }})</td>
+             <td class="text-right">
+              <span class="badge badge-light p-2">{{ diagnostics.FW }} ({{ diagnostics.firmware_short_hash }})</span>
+              </td>
              {% endif %}
            </tr>
            <tr>
              <td><span class="uil uil-focus icon"></span> Variant</td>
-             <td class="text-right">{{ diagnostics.FRIENDLY }}</td>
+             <td class="text-right">
+              <span class="badge badge-light p-2">{{ diagnostics.FRIENDLY }}</span>
+              </td>
            </tr>
            <tr>
              <td><span class="uil uil-shield-check icon"></span> Hardware Serial Number</td>
-             <td class="text-right">{{ diagnostics.serial_number }}</td>
+             <td class="text-right">
+              <span class="badge badge-light p-2">{{ diagnostics.serial_number }}</span>
+              </td>
            </tr>
            <tr>
              <td><span class="uil uil-external-link-alt icon"></span> Hostname</td>
              {% if hostname %}
-             <td class="text-right">{{ hostname }}</td>
+             <td class="text-right">
+              <span class="badge badge-light p-2">{{ hostname }}</span>
+            </td>
              {% else %}
              <td class="text-danger text-right">Unknown</td>
              {% endif %}
            </tr>
            <tr>
              <td><span class="uil uil-cloud-data-connection icon"></span> Local IP Address</td>
-             <td class="text-right">{{ device_info.ip_address }}</td>
+             <td class="text-right">
+              <span class="badge badge-light p-2 mb-1 firstIpAddress"></span>
+              <span class="badge badge-light p-2 mb-1 secondIpAddress"></span>
+              <span class="badge badge-light p-2 mb-1 thirdIpAddress"></span>
+              <span class="badge badge-light p-2 mb-1 fourthIpAddress"></span>
```

Comment:
the only thing I would say here is that there could in theory be an infinite number of IP addresses.

i guess that there probably wouldnt be more than 1 or 2 in most cases, but definitely could be 5 to 10.

Reply:
Sorted!